doc/install_macos.txt

```diff
@@ -0,0 +1,193 @@
+.. include:: css.txt
+
+.. _install_macos:
+
+
+Mac OS Installation Instructions
+################################
+
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda, Homebrew or
+MacPorts but if you did it differently and it worked, please let us
+know the details on the `theano-users`_ mailing-list, so that we can
+add alternate instructions here.
+
+.. _theano-users: http://groups.google.com/group/theano-users?pli=1
+
+In academia: Enthought Canopy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are working in academia, the easiest way to install most of the
+dependencies is to install
+`Canopy <http://enthought.com/products/canopy/>`_.
+If you are affiliated with a university (as student or employee), you
+can download the installer for free.
+
+The Canopy installation includes in particular Python (and the
+development headers), NumPy, SciPy, nose, sphinx, pip, pydot
+(but *not* `Graphviz`_, which is necessary for it to work) and the MKL
+implementation of blas.
+
+
+.. note::
+
+   If you use version 0.6 or later of Theano, we try to automatically
+   link with the Canopy blas version.  Due to Mac OS peculiarities,
+   this requires user intervention.  We detect if the manipulation was
+   done or not and give an error message explaining what to do in case
+   it hasn't been done.
+
+.. include:: install_generic.txt
```

Comment:
For Mac this should go after the homebrew part, as that is an alternative to conda way to install the requirements.